ts/src/program/event.ts

```diff
@@ -89,18 +87,20 @@ export class EventParser {
   private handleSystemLog(log: string): [string | null, boolean] {
     // System component.
     const logStart = log.split(":")[0];
-    // Recursive call.
-    if (logStart.startsWith(`Program ${this.programId.toString()} invoke`)) {
+    // Did the program finish executing?
+    if (
+      logStart.includes("success") ||
+      logStart.match(/^Program (.*) consumed .*$/g) !== null
+    ) {
+      return [null, true];
+      // Recursive call.
```

Comment:
```suggestion
    // Recursive call.
```
Inconsistent spacing.

Reply:
Prettier on crack. Fixed.